php_phongo.c

```diff
@@ -837,38 +859,53 @@ int phongo_execute_command(mongoc_client_t *client, php_phongo_command_type_t ty
 	mongoc_cursor_t *cmd_cursor;
 	zval *zreadPreference = NULL;
 	zval *zsession = NULL;
-	int result;
+	bool result = false;
+	bool free_reply = false;
+	bool free_zsession = false;
```

Comment:
Before you merge this into master, you should run ``clang-format`` on this.

Reply:
This is in the v1.4 branch, which isn't formatted. I intend to run it through clang-format after (or during) the merge to master. Does that work?